libcob/fileio.c

```diff
@@ -4224,7 +4224,7 @@ cob_close (cob_file *f, const int opt, cob_field *fnstatus)
 {
 	char	openMode[OPENMODESIZE];
 
-	memset (openMode, 0, sizeof (openMode));
+	memset (openMode, 129, sizeof (openMode));
```

Comment:
See the reference of memset (https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/memset-wmemset?view=msvc-170)
This line sets all bytes in openMode to 129.
This change may have no effect on the behavior or may change the behavior.
It is not a necessary change, even in resolving the warning message, and should be reverted.